fabric-crash-report-info-v1/src/main/resources/fabric.mod.json

```diff
@@ -16,7 +16,7 @@
     "FabricMC"
   ],
   "depends": {
-    "fabricloader": ">=0.4.0"
```

Comment:
Should also change the parent mod json so it doesn't fail silently.

Reply:
Fabric api itself already depends on 0.13.2.